gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs/services/quick-filters-store.service.ts

```diff
@@ -44,6 +44,7 @@ export class QuickFiltersStoreService {
       perPage,
       from: logFilters.from ? logFilters.from : from,
       to: logFilters.to ? logFilters.to : to,
+      entrypointIds: logFilters.entrypoints ? logFilters.entrypoints.join(',') : undefined,
```

Comment:
I would go with `null` instead of `undefined` to be consistent with following lines. maybe also do a control on length

Reply:
Good idea, I wrongly assumed, that `[]` would evaluate to `false`.